.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - "9"
+script: "npm install && npm test"
```

Comment:
Does adding this cause commits to `arcs-cdn` to gate on tests passing, or is it just informative and we can configure `arcs-cdn` commit behavior separately? If the former we should make sure @sjmiles approves the change.